react/src/components/BAINotificationItem.tsx

```diff
@@ -18,10 +19,14 @@ const BAINotificationItem: React.FC<{
     notification: NotificationState,
   ) => void;
   showDate?: boolean;
-}> = ({ notification, onClickAction, showDate }) => {
+  allowEllipsis?: boolean;
+}> = ({ notification, onClickAction, showDate, allowEllipsis }) => {
   const { t } = useTranslation();
   const { token } = theme.useToken();
   const [showExtraDescription, setShowExtraDescription] = useState(false);
+  const [ellipsis, setEllipsis] = useControllableValue({
+    defaultValue: allowEllipsis,
+  });
```

Comment:
`useControllableValue` helps manage the state to be either self-managed or controlled by its parent. 
`allowEllipsis` is not applicable in this case. `allowEllipsis` determines whether ellipsis is allowed or not. Ellipsis refers to the current state of being ellipsised or not, which is not related to the allowance of ellipsis.

Reply:
Also, you might not need to manage the ellipsis status and don't need to handle it using the `click` event. Typography has a feature to manage it, such as the `expandable` property. Please check [this](https://ant.design/components/typography#components-typography-demo-ellipsis).

Reply:
@yomybaby 

The allowEllipsis property is simply a property that turns ellipsis on/off, so I didn't think it was right to use a useControllableValue, so I removed it. 
The expendable property only provides the ability to expand the omitted text again, not to omit the expanded text again. So in the sider that holds all the notifications, I want to support both collapsing and expanding, so I'm controlling the ellipsis property with a click event.

Reply:
It's okay if it can't use ellipsis again. The main points are:
- The clickable content area is good for the main action. For example, clicking the "success creating session" notification takes you to the session detail page. It's better not to use the clicking action for toggling. Let's reserve clicking for another feature later.
- The UX for toggling ellipsis by clicking again is uncommon and not explicit.

Reply:
Could you let me know what the longest notification you have found is?

Reply:
@yomybaby 
I decided that scalability would be an afterthought, so I modified the code to focus on fixing the mentioned issues.

[ref](https://teams.microsoft.com/l/message/19:14c484402d874dafb15806d093b95a82@thread.skype/1706621647590?tenantId=13c6a44d-9b52-4b9e-aa34-0513ee7131f2&groupId=74ae2c4d-ec4d-4fdf-b2c2-f5041d1e8631&parentMessageId=1706621647590&teamName=devops&channelName=Frontend&createdTime=1706621647590) The error occurs when turn off storage and access the web.  I didn't find the error in my test environment, so I tested it by hardcoding the notification text for the usage calculation part. 

<img width="253" alt="image" src="https://github.com/lablup/backend.ai-webui/assets/51399982/7f6ec185-89c8-4dd5-b555-0f6bd655fe2d">